### ISSUE TITLE
RGB layers only work ENFORCE_BILATERAL is Defined

### ISSUE THREAD

**Issue by keithm999** [sunaku/glove80-keymaps#26]
The RGB layers assigned to the home row keys only work when ENFORCE_BILATERAL is defined.

Removing this definition causes the RGB to stop working for the home row layers, LeftPinky, LeftMiddy, etc. when those keys are held down.

**Comment by sunaku** [comment]
Correct, the RGB illumination patterns are tied to layers in the Glove80 PR30 implementation.  When you disable bilateral combinations, home row mods register modifiers directly (without activating as intermediate layer) and therefore cannot illuminate the per-key RGB patterns.

**Comment by keithm999** [comment]
Thanks for the explanation, that makes sense now.
How much effort do you think it would take to enable the RGB with bilateral disabled? I'm willing to give it a go if you can point me in the right direction to get started.

**Comment by sunaku** [comment]
See @darknao's per-key RGB patch for ZMK at https://github.com/zmkfirmware/zmk/pull/2752 which powers the PR36 beta firmware for the Glove80.  That would be the starting point for implementing the enhancements we have discussed in this issue.  Best of luck! 😎👍  Cheers.
